setup.cfg

```diff
@@ -7,7 +7,7 @@ addopts = --cov django_dramatiq --cov-report term --cov-fail-under=85
 max-line-length = 120
 
 [flake8]
-ignore = E402,F403,F811
+ignore = E402,F403,F811,W503
```

Comment:
Finally i should choose one)

❯ flake8
./tests/test_admin_middleware_race_condition.py:61:68: W504 line break after binary operator
❯ flake8
./tests/test_admin_middleware_race_condition.py:61:13: W503 line break before binary operator